src/frontend/js/components/PaymentButton/index.spec.tsx

```diff
@@ -102,6 +116,10 @@ describe('PaymentButton', () => {
     const product: Joanie.Product = ProductFactory().one();
     const creditCard: Joanie.CreditCard = CreditCardFactory().one();
 
+    fetchMock.get(
```

Comment:
Why is this mock added multiple times ?

Reply:
it's added once by test, it's the call made by `useCurrentOrder`